master/security/service-accounts.md

```diff
@@ -86,6 +86,29 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Restrict label assignment with service account selectors
+
+Network policies can be applied to endpoints using selectors that match labels on either the endpoint itself, the endpoint's namespace, or the endpoint's service account. By specifying selectors based on the endpoint's service account we can employ Kubernetes RBAC to limit which users are allowed to apply labels. In the following example, users with the **intern** role will only be allowed to communicate with endpoints that have that same role.
```

Comment:
I think we should make it more clear it's selecting service accounts in the last sentence.

Maybe something like "pod with an intern service account will only be allowed to communicate with other pods with service accounts labeled "role: intern"

Reply:
Fixed